test/JDBC/expected/BABEL-3395-vu-prepare.out

```diff
@@ -0,0 +1,93 @@
+CREATE TABLE upper_lower_dt (a VARCHAR(20), b NVARCHAR(24), c CHAR(20), d NCHAR(24))
+GO
+INSERT INTO upper_lower_dt(a,b,c,d) values(UPPER(N'Anikait '), LOWER(N'Agrawal '), LOWER(N'Anikait '), UPPER(N'Agrawal '))
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO upper_lower_dt(a,b,c,d) values(UPPER(N' Anikait'), LOWER(N' Agrawal'), LOWER(N' Anikait'), UPPER(N' Agrawal'))
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO upper_lower_dt(a,b,c,d) values(UPPER(N'   A'),LOWER(N'   🤣😃'),LOWER(N'   A'),UPPER(N'   🤣😃'))
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO upper_lower_dt(a,b,c,d) values(LOWER(N' '),UPPER(N' '),UPPER(N' '),LOWER(N' '))
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO upper_lower_dt(a,b,c,d) values(LOWER(N' '),UPPER(N'😊😋😎😍😅😆'),UPPER(N' '),LOWER(N'😊😋😎😍😅😆'))
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO upper_lower_dt(a,b,c,d) values(LOWER(N''),UPPER(N''),UPPER(N''),LOWER(N''))
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO upper_lower_dt(a,b,c,d) values(UPPER(N'a'),LOWER(N'A'),UPPER(N'a'),LOWER(N'A'))
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO upper_lower_dt(a,b,c,d) values(UPPER(NULL),LOWER(NULL),UPPER(NULL),LOWER(NULL))
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO upper_lower_dt(a, b, c, d) values(UPPER(N'比尔·拉'), LOWER(N'比尔·拉'), LOWER(N'比尔·拉'), UPPER(N'比尔·拉'))
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE upper_lower_text(a TEXT)
+GO
+INSERT INTO upper_lower_text values(CAST ('6F9619FF-8B86-D011-B42D-00C04FC964FF' AS text))
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE upper_lower_ntext(a NTEXT)
+GO
+INSERT INTO upper_lower_ntext values(CAST ('6F9619FF-8B86-D011-B42D-00C04FC964FF' AS ntext))
+GO
+~~ROW COUNT: 1~~
+
+
+CREATE TABLE upper_lower_image(a IMAGE)
+GO
+INSERT INTO upper_lower_image values(CAST ('6F9619FF-8B86-D011-B42D-00C04FC964FF' AS image))
+GO
+~~ROW COUNT: 1~~
+
+
+-- UPPER
+CREATE VIEW dep_view_upper AS
+select UPPER(a) AS upper_a, UPPER(b) AS upper_b, UPPER(c) AS upper_c, UPPER(d) AS upper_d from upper_lower_dt WHERE UPPER(a) = N'ANIKAIT' and UPPER(b) = N'AGRAWAL' and UPPER(c) = N'ANIKAIT' and UPPER(d) = N'AGRAWAL';
```

Comment:
should do this testing with different column level collations

Reply:
Done, Added Tests